File: app/src/main/java/com/example/simpleweathermvp/ui/home/HomeView.java
package com.example.simpleweathermvp.ui.home;

import com.example.simpleweathermvp.entity.City;

import java.util.List;

public interface HomeView {

    void showSuggvestionCities(List<City> cities);

    void showCityList(List<City> cities);

    void showError(String message);

    void showEmptyView();
}

File: app/src/main/java/com/example/simpleweathermvp/ui/home/HomeView.kt
package com.example.simpleweathermvp.ui.home

import com.example.simpleweathermvp.domain.entity.City

interface HomeView {
     fun showSuggvestionCities(cities: List<City>)
     fun showListCicies(cities: List<City>)
     fun startWeatherActivity(city: City)

}